applicationLibrary/molecularDynamics/molecularDynamicsLibrary/LJFunctor.h

```diff
@@ -684,6 +737,100 @@ class LJFunctor
+   */
+  size_t getNumFLOPs() {
+    if constexpr (countFLOPs) {
+      size_t numDistCallsAcc = std::accumulate(_aosThreadDataFLOPs.begin(), _aosThreadDataFLOPs.end(), 0ul,
+                                               [](size_t sum, const auto &data) { return sum + data.numDistCalls; });
+      size_t numKernelCallsN3Acc =
+          std::accumulate(_aosThreadDataFLOPs.begin(), _aosThreadDataFLOPs.end(), 0ul,
+                          [](size_t sum, const auto &data) { return sum + data.numKernelCallsN3; });
+      size_t numKernelCallsNoN3Acc =
+          std::accumulate(_aosThreadDataFLOPs.begin(), _aosThreadDataFLOPs.end(), 0ul,
+                          [](size_t sum, const auto &data) { return sum + data.numKernelCallsNoN3; });
+      size_t numGlobalCalcsAcc =
+          std::accumulate(_aosThreadDataFLOPs.begin(), _aosThreadDataFLOPs.end(), 0ul,
+                          [](size_t sum, const auto &data) { return sum + data.numGlobalCalcs; });
+
+      AutoPasLog(TRACE, "Number of Distance Calls           = {}", numDistCallsAcc);
+      AutoPasLog(TRACE, "Number of Newton3 Kernel Calls     = {}", numKernelCallsN3Acc);
+      AutoPasLog(TRACE, "Number of Non-Newton3 Kernel Calls = {}", numKernelCallsNoN3Acc);
+      AutoPasLog(TRACE, "Number of Global Calculations      = {}", numGlobalCalcsAcc);
+
+      constexpr size_t numFLOPsPerDistanceCall = 8;
+      constexpr size_t numFLOPsPerN3KernelCall = 18;
+      constexpr size_t numFLOPsPerNoN3KernelCall = 15;
+      constexpr size_t numFLOPsPerGlobalCalc = applyShift ? 9 : 8;
+
+      return numDistCallsAcc * numFLOPsPerDistanceCall + numKernelCallsN3Acc * numFLOPsPerN3KernelCall +
+             numKernelCallsNoN3Acc * numFLOPsPerNoN3KernelCall + numGlobalCalcsAcc * numFLOPsPerGlobalCalc;
+    } else {
+      autopas::utils::ExceptionHandler::exception("LJFunctor::getNumFLOPs called without countFLOPs enabled!");
+      return 0;
+    }
+  }
+
+  double getHitRate() {
+    if constexpr (countFLOPs) {
+      size_t numDistCallsAcc = std::accumulate(_aosThreadDataFLOPs.begin(), _aosThreadDataFLOPs.end(), 0ul,
+                                               [](size_t sum, const auto &data) { return sum + data.numDistCalls; });
+      size_t numKernelCallsN3Acc =
+          std::accumulate(_aosThreadDataFLOPs.begin(), _aosThreadDataFLOPs.end(), 0ul,
+                          [](size_t sum, const auto &data) { return sum + data.numKernelCallsN3; });
+      size_t numKernelCallsNoN3Acc =
+          std::accumulate(_aosThreadDataFLOPs.begin(), _aosThreadDataFLOPs.end(), 0ul,
+                          [](size_t sum, const auto &data) { return sum + data.numKernelCallsNoN3; });
+
+      AutoPasLog(TRACE, "Number of Distance Calls           = {}", numDistCallsAcc);
+      AutoPasLog(TRACE, "Number of Newton3 Kernel Calls     = {}", numKernelCallsN3Acc);
+      AutoPasLog(TRACE, "Number of Non-Newton3 Kernel Calls = {}", numKernelCallsNoN3Acc);
```

Comment:
I'm slightly in favor of removing these trace calls. If someone is really interested in them `trace` is too verbose to be useful in a "big" simulation, so they would have to implement a dedicated logger or something like that anyways.

Reply:
My reasoning for the `TRACE` is that when debugging, it would be useful to see the "micro-metrics" that lead to the FLOP count + hit rate. In such a case, it would be worth logging these in both functions so that we such "micro-metrics" are available in the future if trying to debug either function. However, as both functions are always currently used together, this is not really needed. This is never intended to be an actual logger, and if this is wanted, a new logger should be created as you suggest.

Reply:
But if I were to debug this, wouldn't I use... the debugger? ;) Especially because first, I'd probably have to recompile the whole project to enable trace logging, so the debugger might be even quicker.
I (think I) get why you say these might be useful. However, I'm not a big fan of keeping things in the code that only "might be" useful if we don't have a clear use case. At the same time this is not a strong opinion so if you prefer it this way keep them.

Reply:
Fair point. Removed in 9321adb5